pyproject.toml

```diff
@@ -75,3 +75,7 @@ explore = [
     "dscribe",
     "scikit-learn",
 ]
+
+torchscript_featurizer = [
```

Comment:
I would name the feature & function `metatensor`, since this is not taking arbitrary TorchScript code. So `pip install chemiscope[metatensor]` and `chemiscope.metatensor_featurizer()`.